src/shogun/preprocessor/RandomFourierGaussPreproc.cpp

```diff
@@ -140,101 +123,66 @@ EFeatureType RandomFourierGaussPreproc::get_feature_type() {
 }
 
 int32_t RandomFourierGaussPreproc::get_dim_feature_space() const {
-	return ((int32_t) dim_feature_space);
+	return ((int32_t) m_dim_feature_space);
 }
 
 void RandomFourierGaussPreproc::set_dim_feature_space(const int32_t dim) {
```

Comment:
actually why not remove the setter?

Reply:
Protest, what if the user has an RFF object and just wants to change dim_feature_space and recompute? , better than creating an entirely new object.